### PR TITLE
[OF-1148] fix: release notes on web

### PR DESCRIPTION
**Pull request by MAG-AW**
connected-spaces-platform Pull Request

**For the reviewer**

* [ ] If required, are the changes covered by appropriate tests?
* [ ] Are any public-facing API changes well documented?
* [ ] Is the code easily readable and extensible and/or follow existing conventions?

### CODE REVIEW

File: teamcity/BuildNPMWebPackage.py [hunk 1]
```diff
@@ -113,6 +115,13 @@ def copy_packages_in(input_args, output_path):
         print("input paths false")
         return False
 
+def copy_readme(input_args, output_path):
+    # Copy readme and create meta file
+    if (os.path.exists(f"{config.default_output_directory}/README.md")):
+        shutil.copy(f"{config.default_output_directory}/README.md", output_path)
+    
+        with open(f"{output_path}/README.md.meta", "w") as f:
```
Comment (MAG-MichaelK) [inline teamcity/BuildNPMWebPackage.py:123]: Meta files are Unity-specific and are not needed for the web package.

**Review by MAG-MichaelK — CHANGES_REQUESTED** [review]

**Review by MAG-MichaelK — APPROVED** [review]

File: teamcity/BuildNPMWebPackage.py [hunk 2]
```diff
@@ -113,6 +115,10 @@ def copy_packages_in(input_args, output_path):
         print("input paths false")
         return False
 
+def copy_readme(input_args, output_path):
+    # Copy readme and create meta file
```
Comment (MAG-AdamThorn) [inline teamcity/BuildNPMWebPackage.py:119]: Remove 'and create meta file' from the comment

**Review by MAG-AdamThorn — CHANGES_REQUESTED** [review]

**Review by MAG-MichaelK — APPROVED** [review]

**Review by Mag-JB — DISMISSED** [review]

File: teamcity/GenerateReadme_NEW.py
[the remark this answers is not in the document]
Comment (MAG-MichaelK) [reply teamcity/GenerateReadme_NEW.py:1]: We should ensure that all commits adhere to our standard. All developers should be using the pre-commit githooks that we include in our repo, unless they explicitly disable them or use the github UI to author a change.

File: teamcity/GenerateReadme_NEW.py [hunk 3]
```diff
@@ -74,13 +74,19 @@ def main():
...
-            continue
-
-        jira_id = match.group("jira_id")
-        tag = match.group("tag")
-        title = match.group("title")
-        is_breaking_change = len(match.group("breaking_change")) > 0
+            match = re.match(r'(?P<jira_id>[a-zA-Z]+\-[0-9]+) (?P<title>.*)', full_title)
+            if match is None:
+                print("Invalid commit title format. Skipping...", file=sys.stderr)
+                continue
```
Comment (MAG-MichaelK) [inline teamcity/GenerateReadme_NEW.py:80]: Can we get a blank line after `continue`, please?

File: teamcity/GenerateReadme_NEW.py [hunk 4]
```diff
@@ -74,13 +74,19 @@ def main():
...
+            jira_id = match.group("jira_id")
+            tag = "misc"
+            title = match.group("title")
+            is_breaking_change = False
```
Comment (MAG-MichaelK) [inline teamcity/GenerateReadme_NEW.py:84]: I don't think we should assume that non-adhering commits are never breaking changes

File: teamcity/templates/readme/readme.md.jinja2 [hunk 5]
```diff
@@ -29,6 +29,5 @@
 | Change ID | Author | Description |
 | --------- | ------ | ----------- |
 {%- for commit in commits %}
-| {{ commit.hexsha | truncate(7, true, '') }} | {{ commit.author }} | {{ commit.message | trim | replace('\n', '<br>') }} |
-{%- endfor -%}
-
+| {{ commit.hexsha | truncate(7, true, '') }} | {{ commit.author }} | {{ commit.message | trim | replace('\r', '') | replace('\n\n', '<br>') | replace('\n', '<br>') | replace('*', '&bull;')}}|
```
Comment (MAG-MichaelK) [inline teamcity/templates/readme/readme.md.jinja2:32]: Why was an explicit replace for `\r` added here? Commit messages should never contain carriage returns by the standard of git.

Also, a space is missing before the last `}}`

**Review by MAG-MichaelK — CHANGES_REQUESTED** [review]

**Review by MAG-MichaelK — APPROVED** [review]

**Review by MAG-SamBirley — APPROVED** [review]

**Pull request merged**